drivers/mspi/mspi_stm32.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2025 EXALT Technologies.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "spi_nor.h"
+
+#ifndef ZEPHYR_DRIVERS_MSPI_STM32_H_
+#define ZEPHYR_DRIVERS_MSPI_STM32_H_
```

Comment:
Nitpicking: to follow the header file path:
```suggestion
#ifndef ZEPHYR_DRIVERS_MSPI_MSPI_STM32_H_
#define ZEPHYR_DRIVERS_MSPI_MSPI_STM32_H_